lib/remote.js

```diff
@@ -113,6 +113,15 @@ Remote.prototype.push = function(refSpecs, opts) {
 };
 
 /**
+ * Lists advertised references from a remote
+ *
+ * @async
+ * @return {Promise<Array<RemoteHead>>} a list of the remote heads currently
+ *                                      available on the remote for download
```

Comment:
"currently available" might be a little off - isn't it more like what the remote had available when the connection was established?